examples/feature_demo/fly_controller_logo.py

```diff
@@ -0,0 +1,155 @@
+"""
+Fly controller logo
+===================
+
+Fly through a cloud of cololoured pygfx logos to inspect the effects of the
+renering.
+"""
+
+# sphinx_gallery_pygfx_docs = 'screenshot'
+# sphinx_gallery_pygfx_test = 'run'
+
+from wgpu.gui.auto import WgpuCanvas, run
+import pygfx as gfx
+import numpy as np
+
+
+custom_sdf = """
+    // Already defined in the shader code.
+    // Maybe we should siloh the shader code in the future.
+    // let m_sqrt_2 = 1.4142135623730951;
+    let m_sqrt_3 = 1.7320508075688772;
+
+    // The coords below assume OpenGL coords, but y is flipped, so we flip the coords.
+    // Its also shifted so that the center of the triangle is at the origin.
+    var coord_for_sdf = vec2<f32>(coord_in.x, -coord_in.y) / size + vec2<f32>(0.5);
+
+    // https://math.stackexchange.com/a/4073070
+    // equilateral triangle has length of size
+    //    sqrt(3) - 1
+    let triangle_x = m_sqrt_3 - 1.;
+    let one_minus_triangle_x = 2. - m_sqrt_3;
+    let triangle_length = m_sqrt_2 * triangle_x;
+
+    let pygfx_width = 0.10;
+
+    let v1 = normalize(vec2<f32>(one_minus_triangle_x, -1));
+    let r1_out = dot(coord_for_sdf, v1);
+    let r1_in  = r1_out + pygfx_width;
+
+    let v2 = normalize(vec2<f32>(-1, one_minus_triangle_x));
+    let r2_out = dot(coord_for_sdf, v2);
+    let r2_in  = r2_out + pygfx_width;
+
+    let v3 = normalize(vec2<f32>(triangle_x, triangle_x));
+    let r3_out = dot(coord_for_sdf - vec2(1, one_minus_triangle_x), v3);
+    let r3_in  = r3_out + pygfx_width;
+"""
+
+inner_triangle_sdf = (
+    custom_sdf
+    + """
+    let inner_offset = 0.5 * (triangle_length - pygfx_width / 2.);
+    let r1_out_blue = -r1_out - inner_offset;
+    let r1_in_blue = r1_out_blue + pygfx_width;
+    let r1_blue = max(
+        max(r2_out, r3_in),
+        max(r1_out_blue, -r1_in_blue)
+    );
+
+    let r2_out_blue = -r2_out - inner_offset;
+    let r2_in_blue = r2_out_blue + pygfx_width;
+    let r2_blue = max(
+        max(r3_out, r1_in),
+        max(r2_out_blue, -r2_in_blue)
+    );
+
+    let r3_out_blue = -r3_out - inner_offset;
+    let r3_in_blue = r3_out_blue + pygfx_width;
+    let r3_blue = max(
+        max(r1_out, r2_in),
+        max(r3_out_blue, -r3_in_blue)
+    );
+
+    return min(r1_blue, min(r2_blue, r3_blue)) * size;
```

Comment:
If this strategy was chosen, it was a little strange to me that the "sdf" had to be multiplied by the size. this would be important to document in `custom_sdf`.

Reply:
do you want to document it or not? feel free to drive this thread to closure yourself by making a decision

Reply:
yeah, i think its useful. took me 1 month to feel comfortable with SDFs